### PR TITLE
:hourglass_flowing_sand:  docstrings: `vectorstores` consistency

### PR DESCRIPTION
**Pull request by leo-gan**
:hourglass_flowing_sand: 
- updated the top-level descriptions to a consistent format;
- changed several `ValueError` to `ImportError` in the import cases;
- changed the format of several internal functions from "name" to "_name". So, these functions are not shown in the Top-level API Reference page (with lists of classes/functions)

@baskaryan BTW Are we going to try [this OpenAI content moderation](https://openai.com/blog/using-gpt-4-for-content-moderation) for our docs?

### CODE REVIEW

**Comment by leo-gan** [comment]
@baskaryan Please, answer me somehow. This message is to test if :arrow_down: this communication channel (PR comments) works or not. 

I have a backlog of unreviewed PRs: 
#9349 
#9333 
#9281 
#9277 
#8651 
#8937 
#8549 
#8398 
#8297

File: libs/langchain/langchain/vectorstores/scann.py [hunk 1]
```diff
@@ -23,9 +22,9 @@ def normalize(x: np.ndarray) -> np.ndarray:
     return x
 
 
-def dependable_scann_import() -> Any:
+def _dependable_scann_import() -> Any:
```
Comment (baskaryan) [inline libs/langchain/langchain/vectorstores/scann.py:25]: all of the dependable...->_dependable... changes are technically breaking changes and imo not worth it (even though in an ideal world i agree they'd be private)

Comment (leo-gan) [reply libs/langchain/langchain/vectorstores/scann.py:25]: Those "import checking" functions are spread all across LC. We even have a generic function for it (but nobody uses it). 
They are really exposing implementation details for users and should not be there in most cases. To me, exposing the internals is more like a bug. 
Anyway, I'll rollback these function names.

**Comment by leo-gan** [comment]
@baskaryan Thank you!

**Pull request merged**